app/constants/payment-request-numbers.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  FIRST_PAYMENT: 1,
+  POST_PAYMENT_ADJUSTMENT: 2
```

Comment:
Probably fine for now but may be a limitation if we have tests that cover three payment requests as `3` is still a post payment adjustment.

Reply:
yup, recognised this myself